app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people.update({name: self})
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_persons_instances = [
+        Person(person["name"],
+               person["age"])
+        for person in people
+    ]
+
+    for i, person in enumerate(people):
+        get_wife = person.get("wife")
+        get_husband = person.get("husband")
+        person_instance = list_of_persons_instances[i]
```

Comment:
You use indexes from one list(people) to take person_instance from another list. 
In my opinion this is not safe, can we be sure in their accordance? May be it's possible to get person_instance in other way?